banzai_nres/utils/phoenix_utils.py

```diff
@@ -0,0 +1,30 @@
+from banzai_nres import settings
+import os
+from astropy.io import fits
+import boto3
+import io
+from banzai_nres.dbs import get_phoenix_model_record
+
+
+def open_model_fits_file(file_location, filename):
+    if 's3' in file_location:
+        s3 = boto3.client('s3')
+        buffer = io.BytesIO()
+        s3.download_fileobj(file_location.replace('s3://', ''), filename, buffer)
+        buffer.seek(0)
+        hdu = fits.open(buffer, memmap=False)
+    else:
+        hdu = fits.open(os.path.join(file_location, filename))
+
+    data = hdu[0].data.copy()
+    return data
+
+
+wavelengths = open_model_fits_file(settings.PHOENIX_WAVELENGTH_FILE_LOCATION, settings.PHOENIX_WAVELENGTH_FILENAME)
+
+
+def load_phoenix_model(db_address, T_effective, log_g, metallicity, alpha):
+    # Load in the template
+    model_record = get_phoenix_model_record(db_address, T_effective, log_g, metallicity, alpha)
```

Comment:
What happens if a set of parameters that do not correspond to a Phoenix model are requested? This need to either fail gracefully, or instead return a model corresponding to the nearest point in the model grid.